includes/class-sensei-lesson.php

```diff
@@ -4782,8 +4782,16 @@ public static function course_signup_link() {
 			$course_link .= esc_html__( 'course', 'sensei-lms' );
 			$course_link .= '</a>';
 
-			// translators: The placeholder %1$s is a link to the Course.
-			$message_default = sprintf( esc_html__( 'Please sign up for the %1$s before starting the lesson.', 'sensei-lms' ), $course_link );
+			$format = '';
+			if ( Sensei_Course::is_self_enrollment_not_allowed( $course_id ) ) {
+				// translators: The placeholder %1$s is a link to the Course.
+				$format = esc_html__( 'Please contact the course administrator to sign up for the %1$s before starting the lesson.', 'sensei-lms' );
```

Comment:
It adds the notice for lessons where the learning mode is not being used.

Reply:
Could we use the same notice that we use everywhere else (i.e. "Please contact the course administrator to access the course content.") for consistency and to reduce the number of new string that need to be translated?